ingest/ingest-config-pipeline.yml

```diff
@@ -24,7 +24,7 @@ trigger:
 
 
 pool:
-  vmImage: 'ubuntu-latest'
+  name: 'amido-stacks-core-data-agent-pool'
```

Comment:
should we add this as variable rather than value directly ?

Reply:
updated with variable